[API 参考](../wiki/Home) / [@142vip/data-source](../wiki/@142vip.data-source) / handlerDataSourceConnectError

# 函数: handlerDataSourceConnectError()

> **handlerDataSourceConnectError**<`T`>(`dataSourceName`, `error`): [`DataSourceParseResponse`](../wiki/@142vip.data-source.%E6%8E%A5%E5%8F%A3.DataSourceParseResponse)<`T`>

定义于: [packages/data-source/src/data-source.utils.ts:27](https://github.com/142vip/core-x/blob/58a4aca72f73ebc92491a458c9b83754486dc296/packages/data-source/src/data-source.utils.ts#L27)

数据源处理错误

## 类型参数

### T

`T`

## 参数

### dataSourceName

`string`

### error

`any`

## 返回

[`DataSourceParseResponse`](../wiki/@142vip.data-source.%E6%8E%A5%E5%8F%A3.DataSourceParseResponse)<`T`>
